### PR TITLE
372 adr renaming wfo

### PR DESCRIPTION
**Pull request by DutchBen**
Implement ADR on component naming

### CODE REVIEW

File: apps/wfo-ui-surf/jest.config.cjs [hunk 1]
```diff
@@ -3,5 +3,5 @@ const base = require('@orchestrator-ui/jest-config/jest-base.config.js');
 module.exports = {
     ...base,
     name: 'wfo-ui-surf',
-    displayName: 'WFO-UI Tests',
+    displayName: 'Wfo-UI Tests',
```
Comment (acidjunk) [inline apps/wfo-ui-surf/jest.config.cjs:6]: Hmm, relying on search and replace?

File: apps/wfo-ui-surf/translations/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ## Adding or overriding translations
 
-The standard WFO component library has translations included, they can be found in messages folder of the @orchestrator-ui/orchestrator-ui-components library. The translations that are provided in this folder (wfo-ui/translations) are merged into these standard translations, this allows for adding to and overriding of the standard translations with the custom ones provided.
+The standard Wfo component library has translations included, they can be found in messages folder of the @orchestrator-ui/orchestrator-ui-components library. The translations that are provided in this folder (wfo-ui/translations) are merged into these standard translations, this allows for adding to and overriding of the standard translations with the custom ones provided.
```
Comment (acidjunk) [inline apps/wfo-ui-surf/translations/README.md:3]: I supposed we can use abbrev in docs.

File: mkdocs/docs/adrs/0005-component-naming.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ Date: 2023-08-08
 *Accepted*
 
 ## Decisions
-To (practically) ensure components have unique names even when they are used in combination with components from other sources we have decided that all components that the @orchesrator-ui package exposes are prefixed with WFO. All complete pages that are exposed are prefixed with WFO and post fixed with Page
+To (practically) ensure components have unique names even when they are used in combination with components from other sources we have decided that all components that the @orchesrator-ui package exposes are prefixed with Wfo. All complete pages that are exposed are prefixed with Wfo and post fixed with Page
```
Comment (acidjunk) [inline mkdocs/docs/adrs/0005-component-naming.md:10]: in 0005 this was WFO

File: package.json [hunk 4]
```diff
@@ -1,8 +1,8 @@
 {
     "private": true,
     "scripts": {
-        "dev": "turbo run dev --filter=wfo-ui...",
-        "dev:surf": "turbo run dev --filter=wfo-ui-surf...",
+        "dev": "turbo run dev --filter=Wfo-ui...",
```
[the remark this answers is not in the document]
Comment (ricardovdheijden) [reply package.json:4]: i think this is accidental @DutchBen , i would say it should match how the app is named in package.json of that specific app

File: apps/wfo-ui-surf/jest.config.cjs (hunk 1 above)
Comment (DutchBen) [reply apps/wfo-ui-surf/jest.config.cjs:6]: Yes. But this renaming is still valid right. We apply the convention in displaynames to no?

File: mkdocs/docs/adrs/0005-component-naming.md (hunk 3 above)
Comment (DutchBen) [reply mkdocs/docs/adrs/0005-component-naming.md:10]: reverted

File: mkdocs/docs/adrs/0007-component-naming.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+TwoWord components will be named like so
+-- ProductBlock
+-- ResourceType
+-- FixedInput
+-- ProductTag
+-- Workflow
+
+All components in the Wfo package are named with the Wfo prefix.
+-- This changes Wfo where ever it's used to Wfo to align with abbr. rules used in the backend
```
[the remark this answers is not in the document]
Comment (DutchBen) [reply mkdocs/docs/adrs/0007-component-naming.md:20]: changed

File: package.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (DutchBen) [reply package.json:4]: reverted

File: apps/wfo-ui-surf/jest.config.cjs (hunk 1 above)
Comment (ricardovdheijden) [reply apps/wfo-ui-surf/jest.config.cjs:6]: The convention should be in filenames, variable names, component names. Strings representing the app name should still be WFO i would say

File: packages/orchestrator-ui-components/src/components/WfoSubscription/WfoRelatedSubscriptions.tsx [hunk 6]
```diff
@@ -78,12 +78,12 @@ export const WFORelatedSubscriptions = ({
...
     const relatedSubscriptionsPageInfo =
         data?.subscriptions.page[0].inUseBySubscriptions.pageInfo;
-    const tableColumns: WFOTableColumns<RelatedSubscription> = {
+    const tableColumns: WfoTableColumns<RelatedSubscription> = {
         subscriptionId: {
             field: 'subscriptionId',
             name: t('id'),
             width: '100',
-            render: (value) => <WFOFirstPartUUID UUID={value} />,
+            render: (value) => <WfoFirstPartUUID UUID={value} />,
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/components/WfoSubscription/WfoRelatedSubscriptions.tsx:86]: Maybe for some other time / PR, but UUID should be Uuid then

**Review by ricardovdheijden — APPROVED** [review]

**Pull request merged**